pkg/crc/machine/kubeconfig.go

```diff
@@ -27,6 +27,20 @@ const (
 	developerContext = "crc-developer"
 )
 
+func updateKubeadminCerts(clientKey, clientCrt []byte, srcKubeconfigPath, destKubeconfigPath string) error {
+	cfg, err := clientcmd.LoadFromFile(srcKubeconfigPath)
+	if err != nil {
+		return err
+	}
+	for username, auth := range cfg.AuthInfos {
+		if username == "admin" {
+			auth.ClientCertificateData = clientCrt
+			auth.ClientKeyData = clientKey
+		}
+	}
+	return clientcmd.WriteToFile(*cfg, destKubeconfigPath)
```

Comment:
if we don't find the admin user that means our source kubeconfig file is corrupted which is generated as part of installer, so I didn't give it much thought about not having the `admin` user but if we go with code logic it will not fail/warning or even create it. In this case it will just dump source kubeconfig to destination kubeconfig path.

Reply:
@cfergeau We should return an error in case `admin` user is not found which means that kubeconfig is not updated properly.

Reply:
I did not see the latest iteration, looks like this could be simplified to:
```
auth, adminUserExists := cfg.AuthInfos["admin"]
if !adminUserExists {
    return fmt.Errorf("...")
}
auth.ClientCertificateData = clientCrt
auth.ClientKeyData = clientKey
```